build.gradle

```diff
@@ -158,11 +159,17 @@ dependencies {
     compile 'net.jcip:jcip-annotations:1.0'
     compile 'com.github.spotbugs:spotbugs-annotations:3.1.12'
 
+    // redefining with version 2.4.1 as applicationinsights-agent-2.5.0 has an open issue
```

Comment:
does 2.5.1-BETA has the issue as well? if not, can we use 2.5.1-BETA here for consistency with the buildfile?

Reply:
no need to redefine them here as java-logging will transitively get 2.5.1-BETA